src/main/java/io/spokestack/spokestack/Spokestack.java

```diff
@@ -197,6 +197,12 @@ public void start() throws Exception {
 
     /**
      * Stops the speech pipeline and releases all its internal resources.
+     *
+     * <p>
+     * This is useful for stopping passive listening (listening for wakeword
+     * activation); for fully releasing <em>all</em> internal resources held by
+     * Spokestack, see {@link #release()}.
```

Comment:
```suggestion
     * the Spokestack pipeline, see {@link #release()}.
```

Reply:
Saying "the Spokestack pipeline" here introduces confusion between Spokestack (a collection of all modules), and the speech pipeline. This is meant to refer to the former.